test/container/detail/random_access_iterator_test.cpp

```diff
@@ -0,0 +1,480 @@
+// ============================================================================
```

Comment:
If the other file is moved to seqan3/range/detail/, the test should be, as well!